src/app/announcement/announcement-detail/announcement-detail.component.html

```diff
@@ -150,14 +154,17 @@
                 tooltipPosition="top"
                 tooltipEvent="hover"
               ></p-calendar>
-              <label for="announcement_detail_form_endDate">{{ 'ANNOUNCEMENT.END_DATE' | translate }}</label>
+              <label id="date-from-label" for="announcement_detail_form_endDate">
+                {{ 'ANNOUNCEMENT.END_DATE' | translate }}</label
+              >
             </span>
           </div>
         </div>
         <div class="col-6 sm:col-2 py-0 sm:pl-4">
           <div class="w-1 flex flex-column row-gap-2 mr-2">
             <p-inputSwitch
               id="announcement_detail_form_status"
+              inputId="announcement_detail_form_status"
```

Comment:
id AND inputId ?
In this case, I think, id is obsolet

Reply:
id is the id for p-inputSwitch. inputId forwards an id to <input> element inside <p-inputSwitch>. Either delete id or rename inputId ?